config.go

```diff
@@ -789,6 +795,30 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	// This value must also be smaller than the largest int on 32-bit
+	// systems.
+	if cfg.MaxPrioTicketSize > 0x7FFFFFFF {
+		str := "%s: maxprioticketsize set too large (must be less than " +
+			"2147483647)"
```

Comment:
this error message is wrong, or the check is wrong.  the number you have in the error is exactly 0x7FFFFFFF.